### PR TITLE
[API] Add missing inStock field to swagger

### PR DESCRIPTION
**Pull request by arti0090**
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | no
| New feature?    | yes (missed thing)
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | Extends #12639
| License         | MIT

I forgot to add this to PR, forgive me sylius 🙏

### CODE REVIEW

**Review by lchrusciel — APPROVED** [review]
forgiveness granted

**Review by GSadee — CHANGES_REQUESTED** [review]
I'm almost sure, that this field has been added to ProductVariant, not Product

**Comment by arti0090** [comment]
Thats true @GSadee i am just changing it

**Comment by arti0090** [comment]
Zrzut ekranu 2021-05-20 o 09 04 11

Changed for ProductVariant, also fixed the missing `price` field

File: src/Sylius/Bundle/ApiBundle/Swagger/ProductVariantDocumentationNormalizer.php [hunk 1]
```diff
@@ -26,12 +26,20 @@ public function normalize($object, $format = null, array $context = [])
...
         $docs = $this->decoratedNormalizer->normalize($object, $format, $context);
 
-        $docs['components']['schemas']['ProductVariant:jsonld-product_variant:read']['properties']['price'] = [
+        $priceSchema = [
             'type' => 'int',
             'readOnly' => true,
             'default' => 0,
         ];
 
+        $docs['components']['schemas']['ProductVariant.jsonld-admin.product_variant.read']['properties']['price'] = $priceSchema;
```
Comment (GSadee) [inline src/Sylius/Bundle/ApiBundle/Swagger/ProductVariantDocumentationNormalizer.php:35]: `price` field has been also added only for shop endpoints
```suggestion
```

**Review by lchrusciel — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, @arti0090! :tada:

**Pull request merged**